.github/workflows/dev-deployment.yml

```
@@ -44,7 +44,9 @@ jobs:
           aws-region: ${{ secrets.AWS_REGION }}
 
       - name: Initialize Terraform
-        run: terraform init
+        run: |
+          cd iac
```

is that the right way to do it?

---

FWIW it might actually be better to use a `working-directory` parameter, like so:

```suggestion
        working-directory: iac
        run: terraform init
```